definitions/npm/moment_v2.x.x/flow_v0.104.x-/moment_v2.x.x.js

```diff
@@ -169,6 +175,12 @@ declare class moment$Moment {
       | void
   ): moment$Moment;
   static utc(string: string, format: string | Array<string>): moment$Moment;
+  static utc(
+    string: string,
```

Comment:
It was a bit odd to see that `moment()` constructor made the first `string` argument be optional with `?string` but `moment.utc` didn't.

Seems like https://github.com/flow-typed/flow-typed/pull/2875 did not do the changes for `moment.utc` constructor. If this is something that we'd want to have in `flow-typed`, I can do that in another PR — but it is not in the scope of this PR, in my opinion.